package.json

```
@@ -20,7 +20,7 @@
     "@guardian/automat-modules": "^0.3.8",
     "@guardian/braze-components": "^4.1.0",
     "@guardian/commercial-core": "0.24.x",
-    "@guardian/consent-management-platform": "6.11.5",
+    "@guardian/consent-management-platform": "9.x.x",
```

Maybe we want to ensure that we require at least 9.0.1, as 9.0.0 is broken!

```suggestion
    "@guardian/consent-management-platform": "^9.0.1",
```